src/router/index.js

```diff
@@ -16,6 +18,16 @@ const routes = [
     name: 'Results',
     component: ResultsContainer,
   },
+  {
```

Comment:
Thanks for doing some research on `eventBus` so we can get this object for `ResultsItemDetails` back in the `routes` array.  Glad this solves out render issue in `App`.

Reply:
Glad we found a viable solution! Completely agree with using Vuex at some point instead of eventBus instead for better scalability.

Reply:
I think we’re in agreement here. Curious to see if eventBus will add challenges in testing. My understanding is that Vuex is not bad to test (but that might be relative to testing redux, not eventBus)